### PR TITLE
#203 test migration to selene 2.x version (part 1)

### PR DESCRIPTION
**Pull request by aleksandr-kotlyar**
### What
Actualize part of integration tests from `tests_from_past/integration` selene v1 to `tests/integration` selene v2. 
Relates #203

### Questions to resolve
- [x] https://github.com/yashaka/selene/pull/237#discussion_r549935924
- [x] https://github.com/yashaka/selene/pull/237#discussion_r549940170

### Review
- [ ] module names - ok
- [ ] test function names - ok
- [ ] error messages in `error_messages_test.py` - ok

### CODE REVIEW

File: tests/integration/element__scroll_to__test.py [hunk 1]
```diff
@@ -22,20 +22,13 @@
...
-    config.browser_name = "chrome"
+def test_can_scroll_to(session_browser):
+    session_browser.open(start_page)
 
+    session_browser.element("#invisible_link").scroll_to().click()
 
-def test_can_scroll_to():
-    open_url(start_page)
-    # logging.warning(driver().current_url)
-    # driver().set_window_size(300, 400)
```
Comment (aleksandr-kotlyar) [inline tests/integration/element__scroll_to__test.py:39]: @yashaka I deleted this code, but want to clarify some questions about it:
```
# logging.warning(driver().current_url)	
# driver().set_window_size(300, 400)
```
Do we need new test for low dimension? 
Or do we need to uncomment this code in this test?

**Review by aleksandr-kotlyar — COMMENTED** [review]
Left some questions to consern.

**Comment by aleksandr-kotlyar** [comment]
> Migrate and actualize tests from `tests_from_past/integration` selene v1 to `tests/integration`. selene v2.
> Relates #203

Ready for review

File: tests/integration/element__find_by_locators__test.py [hunk 2]
```diff
@@ -83,18 +49,19 @@ def test_complex_locator_based_on_by_locators():
...
-    driver.element('#container')\
+    session_browser.element('#container')\
         .element(by.text('Second'))\
         .element(by.be_parent())\
         .element(by.be_following_sibling())\
         .element(by.be_first_child())\
         .click()
-    assert ('second' in driver.current_url) is True
+
+    assert ('second' in session_browser.driver.current_url) is True
```
[the remark this answers is not in the document]
Comment (yashaka) [reply tests/integration/element__find_by_locators__test.py:59]: It looks pretty weird to me too:) 
Of course it shoudl be as simple as `assert 'second' in session_browser.driver.current_url`

File: tests/integration/element__scroll_to__test.py (hunk 1 above)
Comment (yashaka) [reply tests/integration/element__scroll_to__test.py]: we need to uncomment, the idea was to have the browser size small enough so we can't see the needed element and have to scroll to it...

Comment (aleksandr-kotlyar) [reply tests/integration/element__scroll_to__test.py]: Uncommented. Tested - window size is small and scroll works fine. Test passes.
Fixed in https://github.com/yashaka/selene/pull/237/commits/393c6a8e7c80b185df24fdb10003b9f9492eeeaa

File: tests/integration/element__find_by_locators__test.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (aleksandr-kotlyar) [reply tests/integration/element__find_by_locators__test.py]: Fixed in https://github.com/yashaka/selene/pull/237/commits/393c6a8e7c80b185df24fdb10003b9f9492eeeaa
Prettified.

**Pull request merged**